courses/free-courses-hi.md

```diff
@@ -274,6 +274,7 @@
 ### Figma
 
 * [Figma Tutorial](https://www.youtube.com/playlist?list=PLuRPummNMvINdAbI_WT7R5vdjcyRPeRiq) - Pelfizz Studio
+* [Figma Tutorial For Beginners in Hindi](https://www.youtube.com/watch?v=UjPpvf4TNLI) - LearnGrow
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes